source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
```

Comment:
On https://docs.registers.service.gov.uk/linked_registers/ I've used the placeholder `{register}` rather than `{register-name}`: I would recommend changing this to match that or that to match this for consistency's sake.